### DISCUSSION TITLE
Doubt about having the executable in bin/ but not the whole repository.

### DISCUSSION THREAD

**Discussion by CristopherA96** [kovidgoyal/kitty#6032] [Q&A]
Hi,

Is it possible to only copy the launchers (kitty and kitten) to my system (/usr/local/bin) without creating a symbolic link from the repository to my system?

I tried but no success, because when I try to run the tool a python error appears, saying something about __main__.py. So, I set the symbolic links.

I think that the idea of having the complete repository is so that updates to the tool can be done automatically, right? 

**Note:** I built from source code.

**Comment by kovidgoyal** [comment]
No, you have to use symbolic links. And no it has nothing to do with
updates. The executables load their resources based on where they are in
the filesystem.
